redirectPostLogin.php

```diff
@@ -0,0 +1,79 @@
+<html>
+    <head>
+        <title>Member</title>
+        		<script>
+	function gotologin() {
+    location.href = "login.php";
+}
+
+</script>
+    </head>
+    <body bgcolor=#b2b2b2>

```

Comment:
For future reference, if you want to set a background color in html, you need quotations around the color, but it should be in the css anyways :)